### ISSUE TITLE
Can I config my loss function without change source code?

### ISSUE THREAD

**Issue by ZYG151541155** [ThilinaRajapakse/simpletransformers#1235]
**Is your feature request related to a problem? Please describe.**
Hi, when I dual with multi-class text classification data, because of  data unbalanced, and I want to change my loss function from CrossEntropyLoss to focal loss or dice loss. But I don't know whether can I config my loss fuction? 

**Describe the solution you'd like**
I check simpletranformers.ai tutorials, But I don't find the answer to fit my question, Can you help me or give me some suggestion?? thaks！

**Comment by Zhylkaaa** [comment]
Maybe there will be some people with the same question, so I will cross post this conversation in pull request #1209 
Request is not merged and stays under development, but I encourage anyone to try it and contribute if you want :)
